lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -112,6 +124,37 @@ class AwsCompileKafkaEvents {
       functionObj.events.forEach((event) => {
         if (!event.kafka) return;
 
+        if (!event.kafka.accessConfigurations || event.kafka.accessConfigurations.length === 0) {
```

Comment:
I think we can assume that `event.kafka.accessConfigurations` will be defined because it is marked as required on schema level and I think nothing really changed about that in scope of this PR, right?

Reply:
removed 😄